.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -57,11 +69,17 @@ jobs:
       run: |
         set -eu
         since_time=$(date -u -d '24 hours ago' '+%a, %d %b %Y %H:%M:%S GMT')
+        if [ "$JOB" = "nvd-api" ]; then
+            url="https://definitions.stackrox.io/v4/nvd/nvd-api.zip"
+          else
+            url="https://definitions.stackrox.io/v4/nvd/nvd-feeds.zip"
+          fi
+          
         curl \
-            -o nvd.zip \
-            -w "%{http_code}" \
-            -H "If-Modified-Since: $since_time" \
-            https://definitions.stackrox.io/v4/nvd/nvd.zip \
+          -o nvd.zip \
+          -w "%{http_code}" \
+          -H "If-Modified-Since: $since_time" \
+          $url \
         | grep 200
```

Comment:
Would you be up for logging the status code as well, like this:


```sh
        code=$(curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            "$url")

        echo "code: $code"
        echo "$code" | grep -q 200
```